ethereum/gnosis_protocol_v2/solver_names.csv

```diff
@@ -0,0 +1,56 @@
+address,environment,name
```

Comment:
0xCa0b051804f57412600C2597d62E688d813694Ba, Ethereum, holytrade

Reply:
What is this all about?